spec/repository_spec.rb

```diff
@@ -26,3 +26,17 @@
     expect(EXTRA_FILES).to eq([])
   end
 end
+
+describe "The file \"renamed-styles.json\"" do
+  it "must be valid JSON" do
+    expect(JSON_IS_VALID).to be true
+  end
```

Comment:
Replace `CSL.validate(path)` by `CSL::Style.load(path)` gives

```
Finished in 45.12 seconds (files took 39 seconds to load)
123791 examples, 0 failures
```

so maybe the savings would only be 5-10 seconds here, since I came to 38 seconds without any validation.